Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift

```diff
@@ -172,6 +176,16 @@ public final class HTTP2StreamMultiplexer: ChannelInboundHandler, ChannelOutboun
         context.fireUserInboundEventTriggered(event)
     }
 
+    public func channelWritabilityChanged(context: ChannelHandlerContext) {
+        let newWritability = context.channel.isWritable
+
+        for channel in self.streams.values {
+            channel.parentChannelWritabilityChanged(newValue: newWritability)
```

Comment:
I think it is not save to use the `newWritability` here as this value may change in between. General speaking this method may even re-entrance and so it would lead to out-of-order notifies when using `newWritability` here.